todoman/ui.py

```diff
@@ -251,9 +251,23 @@ def detailed(self, todo):
         return rv
 
     def format_date(self, date):
+        """
+        Returns date in the following format:
+        * if date == today or tomorrow: "Today" or "Tomorrow"
+        * else: return a string representing that date
+        * if no date is supplied, it returns empty_date
+        """
         if date:
-            rv = date.strftime(self.date_format)
-            return rv
+            date_tomorrow = self.now.date() + timedelta(days=1)
+            # Get a dictionary which stores the dates for which
+            # we don't have to merely return a date string and
+            # map it to the special string we need to return
+            WIDTH = len(date.strftime(self.date_format))
```

Comment:
Also, since ```self.empty_date``` calls ```self.format_date(self.now))```, it would not be an empty_date, rather something like "        Today" (ideally - when we can get the code to run) and I think it would hinder the utility of having an empty_date and therefore would render it useless! :(